---
layout: default
title: Data
permalink: /data/
---

Data availability

- [HMP reference genomes](https://www.hmpdacc.org/hmp/catalog/) for Gastrointestinal tract (accessed March 18, 2018; 823 genome sequences)

Gastrointestinal_tract.nuc.fsa downloaded from [here](https://www.hmpdacc.org/hmp/HMRGD/) - the FASTA (ASM) format chosen.


 The HMP's intial aim was to sequence 3000 genomes from both cultured and uncultured bacteria, plus several viral and small eukaryotic microbes isolated from human body sites. This, in conjunction with reference genomes sequenced by HMP Demonstration Projects and other members of the International Human Microbiome Consortium (IHMC), were to supplement the available selection of non-HMP funded human-associated reference genomes to provide a comprehensive pool of genome sequences to aid in the analysis of human metagenomic data.

Sequencing efforts were largely bacterial-focused, however a smaller number of viruses, archaea and small eukaryotic references were included. Reference genomes were selected from the same major body sites as HMP metagenomic sampling efforts, as well as additional body sites not targeted by HMP metagenomic sampling, to provide the most thorough representation.

 The majority of the Microbial Reference Genomes were sequenced only to a high-quality draft stage. High-quality draft sequences do not include every base of the genome, rather they are assemblies of several large contiguous pieces of sequence (contigs) with subsequent gaps in sequence knowledge. Nonetheless, they provide enough information for a general assessment of gene content. About 15% of the Microbial Reference Genomes were taken to improved levels of finishing up to and including a finished (often referred to as Gold Standard) genome.
The community-defined genome finishing standards adopted by the HMP were aggregated from the following references:

    P. S. G. Chain, et al., Genome Project Standards in a New Era of Sequencing. Science 9 October 2009: Vol. 326. no. 5950, pp. 236 - 237
    The Human Microbiome Jumpstart Reference Strains Consortium, A Catalog of Reference Genomes from the Human Microbiome.
    Science 21 May 2010:Vol. 328. no. 5981, pp. 994 - 999


Metadata pertaining to all reference genomes, including body site, finishing status and links to NCBI and public repositories, can be found in the HMP Project Catalog. Sequence data is available through the legacy DACC Data Browser, or through NCBI BioProject 28331.

Reference genome efforts were coordinated through the HMP Strains Working Group. This working group included representatives of the HMP Sequencing Centers, DACC, Technology Development grantees, and a group of body site experts providing feedback and recommendations on strain selection. Reference genomes were collected from public repositories, through collaborations with sequencing centers, and via community feedback.

Analyses describing the initial set of HMP reference genomes were published in 2010:

    Human Microbiome Jumpstart Reference Strains Consortium., A catalog of reference genomes from the human microbiome. Science 21 May 2010: 328(5981), pp. 994-97.


 The HMP plans to sequence, or collect from associated efforts, a total of 3000 reference genomes isolated from human body sites. The majority of these will be sequenced only to a high-quality draft stage. Metadata about current, completed and targeted reference genome projects can be found in the Project Catalog. The information gained from the Reference Genomes will aid in taxonomic assignment and functional annotation of 16S RNA and metagenomic sequence, respectively, from microbiome samples.

As reference genomes are released with annotation, they will become available for download here. This page does not reflect every project found in the HMP Project Catalog, but only those that have completed sequencing and annotation. Isolates are organized by body site, then by genus and species name. Users can sort within body site by Genbank project id. Hover over download icons to see file format type and file size. The DCC provides the following four file formats: assembly nucleotide fasta (ASM), protein multifasta (PEP), coding sequence nucleotide multifasta (CDS), and genbank format (GBK). Sequence and annotation data is also available at NCBI.

This page is updated monthly. Contact the DCC if you need access to a previous version of the dataset.

HMP BLAST Server - BLAST against all available HMP Reference Genomes or body site specific subsets

[Databrowser](https://www.hmpdacc.org/hmp/resources/data_browser.php)

Or query: https://www.hmpdacc.org/hmp/catalog/grid.php?dataset=genomic&hmp_isolation_body_site=gastrointestinal_tract

Could also look at https://www.ncbi.nlm.nih.gov/bioproject/28331


References

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3779803/
http://meta.genomics.cn/meta/dataTools
https://www.hmpdacc.org/hmp/resources/osdf.php
https://portal.hmpdacc.org/
https://www.nature.com/articles/sdata201735
